docker/snapserver/snapserver-example.conf

```diff
@@ -73,7 +73,8 @@ doc_root = /usr/share/snapserver/snapweb/
 # stream URI of the PCM input stream, can be configured multiple times
 # Format: TYPE://host/path?name=NAME[&codec=CODEC][&sampleformat=SAMPLEFORMAT]
 #stream = pipe:///tmp/snapfifo?name=default
-stream = pipe:///tmp/snapfifo?name=Default&sampleformat=48000:16:2&controlscript=meta_mopidy.py
+stream = pipe:///tmp/snapfifo?name=Mopidy&sampleformat=48000:16:2&controlscript=meta_mopidy.py
+stream = librespot:///librespot?name=Spotify

```

Comment:
Means that out-of-the-box, Snapcast server will accept streams from Spotify Connect. Requires use of snapserver > [1.1.0](https://github.com/jaedb/snapserver/releases/tag/1.1.0)